tests/unit/makefile

```diff
@@ -243,15 +271,22 @@ ifeq ($(MFEM_USE_CUDA),YES)
 	@$(call mfem-test,$<,, CEED Unit tests (cuda-gen),--device ceed-cuda:/gpu/cuda/gen $(MFEM_DATA_FLAG),SKIP-NO-VIS)
 endif
 
+mjit_tests-test-seq: mjit_tests
+	@$(call mfem-test,$<,, Mjit tests,--device cpu,SKIP-NO-VIS)
+
 RUN_MPI = $(MFEM_MPIEXEC) $(MFEM_MPIEXEC_NP)
 %-test-par: %
 	@$(call mfem-test,$<, $(RUN_MPI) 1, Parallel unit tests,$(MFEM_DATA_FLAG),SKIP-NO-VIS)
 	@$(call mfem-test,$<, $(RUN_MPI) $(MFEM_MPI_NP), Parallel unit tests,$(MFEM_DATA_FLAG),SKIP-NO-VIS)
 
+pmjit_tests-test-par: pmjit_tests
+	@$(call mfem-test,$<, $(RUN_MPI) 1, Parallel mjit tests,--device cpu,SKIP-NO-VIS)
+	@$(call mfem-test,$<, $(RUN_MPI) $(MFEM_MPI_NP), Parallel mjit tests,--device cpu,SKIP-NO-VIS)
+
 # Generate an error message if the MFEM library is not built and exit
 $(MFEM_LIB_FILE):
 	$(error The MFEM library is not built)
 
 clean:
 	rm -f $(SEQ_UNIT_TESTS) $(PAR_UNIT_TESTS) *.o */*.o */*~ *~
-	rm -rf *.dSYM output_meshes
+	rm -rf *.dSYM output_meshes libmjit.*
```

Comment:
On Mac, there two files that are not cleaned properly: `libmjit_test_par.dylib` and `libmjit_test_seq.dylib` so I think we want to replace `libmjit.*` with `libmjit*`.

Reply:
`done`